hail/src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -901,27 +901,17 @@ class RVD(
 
   def orderedLeftJoinDistinctAndInsert(
     right: RVD,
-    root: String,
-    lift: Option[String] = None,
```

Comment:
Could you give me some background on this change? Why are we removing lift?

Reply:
this stuff was used long ago when table foreign-key joins were IR, not implemented in Python.